### PR TITLE
New platform automation category

### PR DESCRIPTION
**Pull request by Pilargit12**
- Moved pages from PR #222 based on new build from tech docs

- Added to left navigation panel and added category to home page options

- Still need to review document sending this as a draft since it's not ready for review

### CODE REVIEW

**Comment by Pilargit12** [comment]
@StevenBarre - feel free to delete https://github.com/bcgov/platform-developer-docs/pull/222

Let's work on this PR!

**Comment by mtspn** [comment]
Should we consider changing at the naming of the left nav sections if they're going to be changed to alphabetical order- presumably for easier navigation? 

For example:
App monitoring = monitoring
Platform architecture reference = architecture reference 
OpenShift projects and access = projects and access

**Comment by Pilargit12** [comment]
Hi @mtspn I just made the change of the categories to be in alphabetical order in the latest build (you can see it now live on tech docs only), but I did not think of shortening the names a nor re-organizing the categories as I am hoping at some point to do user testing once in backstage.

File: src/pages/index.js [hunk 1]
```diff
@@ -236,6 +236,20 @@ const IndexPage = ({ location }) => {
...
+            <ul>
+              <li>
+              <Link to={"/platform-automation/"}>Platform automation</Link>  
+              </li>
+              <li>
+              <Link to={"/alertmanager/"}>AlertManager</Link>
+              </li>
+              <li>
+               <Link to={"/automated-scaling/"}>Automated scale-down</Link> 
+              </li>
```
Comment (StevenBarre) [inline src/pages/index.js:250]: Should these be re-ordered to match the `sort_order` in the markdown files?

Comment (Pilargit12) [reply src/pages/index.js:250]: The sort order in the .md files on the left navigation are determined based on the number inside the files under the content_owner section. 

For the order in the homepage section, in index.js - we determine the order in which they are placed.

Comment (mtspn) [reply src/pages/index.js:250]: I agree that it makes sense to match the `sort_order` and the order listed in the homepage defined on index.js

**Review by mtspn — APPROVED** [review]

**Review by StevenBarre — APPROVED** [review]

**Review by mtspn — APPROVED** [review]

**Pull request merged**